ansible/olm.yaml

```diff
@@ -21,60 +25,35 @@
   - name: Download opm
     get_url:
       url: https://github.com/operator-framework/operator-registry/releases/{{ opm_url_suffix }}/linux-amd64-opm
-      dest: /usr/local/bin/opm
-      mode: '0755'
-
-- hosts: convergence_base
-  become: yes
-  become_user: ocp
-  vars_files: "vars/default.yaml"
-  environment:
-    PATH: "/usr/local/bin:{{ ansible_env.PATH }}"
-    KUBECONFIG: "{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
+      dest: "{{ working_bin_dir }}/opm"
+      mode: 0755
 
-  tasks:
-  - name: Create {{ base_path }}/github.com/openstack-k8s-operators
-    file:
-      path: "{{ base_path }}/github.com/openstack-k8s-operators/openstack-cluster-operator"
-      state: directory
-      mode: '0755'
+  - set_fact:
+      repo_dir: "{{ k8s_operators_dir }}/openstack-cluster-operator"
 
   - name: Clone openstack-k8s-operators with force=yes to be able to rerun
     git:
-      repo: "{{ openstack_k8s_operators_https_url | default('https://github.com/openstack-k8s-operators', true) }}/{{ item }}.git"
-      dest: "{{ base_path }}/github.com/openstack-k8s-operators/{{ item }}"
-      version: "{{ openstack_k8s_operators_cluster_branch | default('HEAD', true) }}"
+      repo: "{{ base_url }}/openstack-cluster-operator.git"
+      dest: "{{ repo_dir }}"
+      version: "{{ branch }}"
       force: yes
-    with_items:
-      - openstack-cluster-operator
-
-  - name: switch to openstack project/namespace
-    ignore_errors: true
-    shell: >
-      oc project openstack
-    register: openstack_project_switch
-
-  # if OpenStack project/namespace does not exist
-  - name: create OpenStack project/namespace
-    shell: |
-      oc adm new-project openstack &&
-      oc project openstack
-    when: openstack_project_switch.rc == 1
+    vars:
+      base_url: "{{ openstack_k8s_operators_https_url | default('https://github.com/openstack-k8s-operators', true) }}"
+      branch: "{{ openstack_k8s_operators_cluster_branch | default('HEAD', true) }}"
 
   - name: build the csv-merger
     shell: >
       make csv-merger
     args:
-      chdir: "{{ base_path }}/github.com/openstack-k8s-operators/openstack-cluster-operator"
-      creates: "{{ base_path }}/github.com/openstack-k8s-operators/openstack-cluster-operator/build/_output/csv-merger"
+      chdir: "{{ repo_dir }}"
 
+  # XXX: This may not rebuild correctly as 'creates' only tests existence.
```

Comment:
The comment here doesn't quite tell the whole story as even if you used a Makefile the csv_version would still be the same. The intent here is to encourage you to bump csv_version if you are building a new OLM catalog and bundle. So lets do just that... "TODO: fail if this directory exists with an error message informing the user to bump the csv_version"